index.bs

```diff
@@ -877,6 +892,15 @@ Each {{XRBoundedReferenceSpace}} has a <dfn for="XRBoundedReferenceSpace">native
 
 The <dfn attribute for="XRBoundedReferenceSpace">boundsGeometry</dfn> attribute is an array of {{DOMPointReadOnly}}s such that each entry is equal to the entry in the {{XRBoundedReferenceSpace}}'s [=XRBoundedReferenceSpace/native bounds geometry=] premultiplied by the {{XRRigidTransform/inverse}} of the [=XRSpace/origin offset=]. In other words, it provides the same border in {{XRBoundedReferenceSpace}} coordinates relative to the [=XRSpace/effective origin=].
 
+
+<div class="algorithm" data-algorithm="bounded-space-supported">
```

Comment:
@thetuvix are you on board with this language?  If not, please feel free to propose an alternative when you update your tracking loss PR.

Reply:
This seems like a reasonable approach.  If the device can produce a floor-based coordinate system, but doesn't know boundaries, it should only support `local-floor`.

If the device somehow knows boundaries without knowing the floor, I suppose the UA could estimate the floor as with `local-floor`, but I don't know of any such devices today, so that would be more theoretical.  We can always relax this later.